.taskcluster.yml

```diff
@@ -95,7 +96,11 @@ tasks:
           && cd  /go/src/github.com/taskcluster/taskcluster-cli
           && git fetch {{ event.head.repo.url }} {{ event.version }}
           && git checkout FETCH_HEAD
-          && make prep release upload
+          && make prep release
+      artifacts:
+        public:
+          path: /go/src/github.com/taskcluster/taskcluster-cli/taskcluster/build
+          type: directory
```

Comment:
.. but it's redundant, and likely to be forgotten when adding a new platform, in which case we'd have missing artifacts anyway (unless the worker is using AI to say "I bet you meant this file too")